leda/components/Collapsible/types.ts

```diff
@@ -1,26 +1,26 @@
 import * as React from 'react';
 
 export interface TransitionOptions {
-  /** Длительность анимации коллапса. Число в миллисекундах. Строки с единицами измерения */
+  /** Aimation duration. A number of milliseconds or a string containing units: '200ms' or '0.5s' */
   duration: number | string,
-  /** Функция анимации коллапса. См. https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function */
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function */
   animation: string,
-  /** Длительность задержки перед началом анимации. Число в миллисекундах. Строки с единицами измерения */
+  /** A delay before animation. A number of milliseconds or a string containing units: '200ms' or '0.5s' */
   delay?: number | string,
 }
 
 export interface CollapsibleProps extends React.HTMLAttributes<HTMLDivElement> {
-  /** Состояние элемента открыт/закрыт */
+  /** Or not */
```

Comment:
You know if will be shown in vs code popup separately, right?))
```suggestion
  /** Open or not */
```

Reply:
Not that fun =^..^=

Reply:
yes and it will be shown like: 
isOpen

or not